packages/extension/public/css/daily-companion-app.css

```diff
@@ -8,4 +8,7 @@ daily-companion-app {
   position: fixed;
   z-index: 2147483647;
   font-feature-settings: normal!important;
+  text-rendering: optimizeLegibility;
```

Comment:
You already have these changes in other PR, but I will repeat my comment - let's insert space before !important